Chap_API_Struct.tex

```diff
@@ -4526,7 +4526,7 @@ \subsection{Job control attributes}
 
 %
 \declareAttribute{PMIX_JOB_CTRL_ID}{"pmix.jctrl.id"}{char*}{
-Provide a string identifier for this request.
+Provide a string identifier for this request. The user can provide an identifier for the requested operation, thus allowing them to later request status of the operation or to terminate it. The host, therefore, shall track it with the request for future reference.
 }
```

Comment:
Should we add a note to PMIX_JOB_CTRL_CANCEL that specified request references a previous request marked with PMIX_JOB_CTRL_ID?

Reply:
Yeah, that makes sense - will do!

Reply:
Done